dolphin/tests/api/v1/test_storages.py

```diff
@@ -0,0 +1,46 @@
+import sys
+import pytest
+import unittest
+
+from dolphin import context
+from dolphin import exception
+
+sys.modules['dolphin.cryptor'] = unittest.mock.MagicMock()
+from dolphin.api.v1.storages import StorageController
+from dolphin.task_manager.tasks import task
+
+
+class Request:
+    def __init__(self):
+        self.environ = {'dolphin.context': context.RequestContext()}
+
+
+class TestStorageController:
+    def test_delete(self, mocker):
+        # For StorageController.__init__
+        mocker.patch('dolphin.task_manager.rpcapi.TaskAPI.__init__', return_value=None)
+        # For StorageController.delete
+        mock_remove_storage_resource = mocker.patch('dolphin.task_manager.rpcapi.TaskAPI.remove_storage_resource')
+
+        sc = StorageController()
+        req = Request()
+
+        """
+        Get storage successfully, call remove_storage_resource
+        Call count depends on the StorageResourceTask's subclasses' count
+        """
```

Comment:
donot use """ """ in the middle of function.

Reply:
Done